README.md

```diff
@@ -4,221 +4,501 @@
-Common write parameters:
-* dfs - list of Dataframes to write
-* index_cols=None (optional) - specify specific index columns, by default Dataframe's index columns will be used.
-* labels=None (optional)
-* max_in_message=0 (optional)
-* partition_keys=None (Not yet supported)
+<a id="method-write-backend-common-params"></a>
+#### Common write Parameters
+
+All Frames backends that support the `write` method support the following common parameters, which can be set in the `attrs` method parameter:
+
+- <a id="method-write-param-dfs"></a>**dfs** &mdash; list of DataFrames to write.
+- <a id="method-write-param-index_cols"></a>**index_cols** (Optional) (default: none) &mdash; specify specific index columns, by default DataFrame's index columns will be used.
+- <a id="method-write-param-labels"></a>**labels** (Optional) (default: none)
+- <a id="method-write-param-max_in_message"></a>**max_in_message** (Optional) (default: `0`)
+- <a id="method-write-param-partition_keys"></a>**partition_keys** (Optional) (default: none) (**Not yet supported**)
 
 Example:
 ```python
-data = [['tom', 10], ['nick', 15], ['juli', 14]] 
-df = pd.DataFrame(data, columns = ['name', 'age'])
-df.set_index('name')
-client.write(backend='kv', table='mytable', dfs=df)
+data = [["tom", 10], ["nick", 15], ["juli", 14]]
+df = pd.DataFrame(data, columns = ["name", "age"])
+df.set_index("name")
+client.write(backend="kv", table="mytable", dfs=df)
 ```
 
-#### backend specific parameters
-##### KV
+<a id="method-write-params-kv"></a>
+#### kv Backend write Parameters
+
 <!--
-* expression=' ' (optional) - for detailed information on update expressions see [docs](https://www.iguazio.com/docs/reference/latest-release/expressions/update-expression/)
+- <a id="method-write-kv-param-expression"></a>**expression** (Optional) (default: none) &mdash; a platform update expression that determines how to update the table.
+  For detailed information about platform update expressions, see the [platform documentation](https://www.iguazio.com/docs/reference/latest-release/expressions/update-expression/).
 -->
   <!-- [IntInfo] [c-no-update-expression-support] (24.9.19) See Bug IG-12510,
     Requirement IG-5339, & DOC IG-12272. The original write example below had
     an `expression` parameter:
-v3c.write(backend='kv', table='mytable', dfs=df, expression='city="NY"', condition='age>14')
+v3c.write(backend="kv", table="mytable", dfs=df, expression="city='NY'", condition="age>14")
     -->
-* condition=' ' (optional) - for detailed information on condition expressions see [docs](https://www.iguazio.com/docs/reference/latest-release/expressions/condition-expression/)
+  <!-- [IntInfo] [c-no-update-expression-support] (24.9.19) See Bug IG-12510,
+    Requirement IG-5339, & DOC IG-12272. -->
+
+- <a id="method-write-kv-param-condition"></a>**condition** (Optional) (default: none) &mdash; a platform condition expression that defines conditions for performing the update.
```

Comment:
Also we support `Expression` (iv'e noticed we didnt document this before, not sure if it was on purpose).
Basicaaly the `expression` is just a regular update expression.

Reply:
We did document `expression` and as part of my recent changes I commented-out this documentation because Golan and Adi decided that we're not ready to support update expressions. See the internal `[c-no-update-expression-support]` comments that I added in the README when I filtered out the update-expression doc.